src/components/CourseDetailsComponent/SubComponents/AboutElements/LearningInfo.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import {
+  Box,
+  Grid,
+  makeStyles,
+  Typography,
+  ListItem,
+  ListItemAvatar,
+  Avatar,
+  ListItemText,
+} from "@material-ui/core";
+
+import CheckIcon from "@material-ui/icons/Check";
+
+const listElement = [
+  "Gain skills required to succeed in an entry-level IT job",
+  "Learn to perform day-to-day IT support tasks including computer assembly, wireless networking, installing programs, and customer service",
+  "Learn how to provide end-to-end customer support, ranging from identifying problems to troubleshooting and debugging",
+  "Learn to use systems including Linux, Domain Name Systems, Command-Line Interface, and Binary Code",
+];
+function LearningInfo() {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.root}>
+      <Box className={classes.container}>
+        <Typography variant="h4" color="textSecondary" gutterBottom>
+          What you will learn
+        </Typography>
+        <Box mt={2} mb={2}>
+          <Grid container item xs={12} spacing={2}>
+            {listElement.map((items, index) => (
+              <>
```

Comment:
Whenever you are mapping something make sure to provide keys.

Reply:
Sure @Abhishek-kumar09   :)